### ISSUE TITLE
Footnote jumps to the next page

### ISSUE THREAD

**Issue by ajluk** [openstax/cnx-recipes#3548]
Footnote number 3 is pushed to the next page:
![image](https://user-images.githubusercontent.com/55203598/108877420-83022c00-75ff-11eb-9d85-712774269999.png)

Based on the comment: https://github.com/openstax/cnx-recipes/issues/3481#issuecomment-775289750
collection ID: col12087
server: production
chapter/section: 21.1
surrounding text: the most common residents in the healthy

**Comment by lwtchu** [comment]
note: this can fall to e.e if needed

**Comment by ajluk** [comment]
I have checked this issue once again after Natalia's changes for footnotes removing additional margin, but the problem still persits: 
![image](https://user-images.githubusercontent.com/55203598/109969290-07396b00-7cf4-11eb-8fd5-4e057cc011fc.png)

**Comment by ajluk** [comment]
This problem need to be checked again after fix for this issue ( #2114) will be merged.

**Comment by MartaSztuka** [comment]
@ajluk Footnotes still breaks
![Screenshot from 2021-03-25 14-47-59](https://user-images.githubusercontent.com/47241114/112484824-92999f80-8d7a-11eb-9678-dc377c19e2c2.png)

**Comment by lwtchu** [comment]
@ajluk do you know what else is needed for this :eyes:

**Comment by ajluk** [comment]
@lwtchu I noticed this problem still persists, but only if there many footnotes appearing on the same page. I'm guessing here Prince is forcing the third footnote to jump to the next page to prevent them to take too much space if they will be put together on on the same page.There is a Prince property that can be used to force a footnote to be on the exact same page as a footnote call, but it can be used only on `paragraph` containing the footnote, not footnote element itself. In the current markup the footnotes in our books are located in separate tags (`aside`), which are not putted in paragraphs. I have written about it under the previous issue regarding footnotes in this comment: https://github.com/openstax/cnx-recipes/issues/3481#issuecomment-781329346  I'm not sure how this can be fixed, I'm afraid. There was a huge work done on the footnotes markup so far as I remember, but I wasn't participate in it, so I'm not sure why the footnotes markup was changed exactly and if it can be modified to fix this issue. :disappointed:
